crates/rustc_codegen_spirv/src/linker/mod.rs

```diff
@@ -136,7 +137,11 @@ pub fn link(sess: &Session, mut inputs: Vec<Module>, opts: &Options) -> Result<M
 
     let mut output = if opts.structurize {
         let _timer = sess.timer("link_structurize");
-        structurizer::structurize(sess, output)
+        if false {
```

Comment:
`if env::var("OLD_STRUCTURIZER").is_ok()` >:)

or replace the old structurizer, not sure if this is intended to be a replacement or not?

Reply:
I mean it looks like this is better in everyway might as well delete my work? or rename mine to `old_structurizer` and this new one to `structurizer` if you want compare it if we run into issues with the new one.